db/migrate/20230221194145_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
```

Comment:
- [ ] Please follow the Ruby name-convenient to be `comments_counter` instead. Also with the `likes_counter` 

_Kindly apply the changes in all similar situations_